cellprofiler_core/reader/__init__.py

```diff
@@ -21,6 +21,10 @@ def add_reader(reader_name, check_svn, class_name=None):
         else:
             cp_reader = find_cp_reader(rdr)
         name = cp_reader.reader_name
+    except ModuleNotFoundError as e:
+        LOGGER.warning("Could not load google.cloud package (is it installed?)")
+        bad_readers.append((reader_name, e))
+        return
```

Comment:
As a minor point - we should not be putting specific warnings here. I anticipate that a few readers could produce ModuleNotFoundErrors, so it's best to either fill the module and reader names in the warning or test the import within the reader class itself.

Reply:
Ah, yeah thanks for catching that.